doc/state-component/api.md

```diff
@@ -0,0 +1,123 @@
+
+Questions :
```

Comment:
Seems ok for me, except that in my opinion our states should not be Views but some kind of data containers, defined in a agnostic way in a specific file. Those data container could then be used into Backbone Views, but also another framework in the future.

What do you thing @misstick @m4dz @CPatchane ?